.github/workflows/ci.yml

```
@@ -1,300 +1,77 @@
+        options: >-
+          --health-cmd "pg_isready -U postgres -d postgres -h 127.0.0.1 -p 5432"
+          --health-interval 10s --health-timeout 5s --health-retries 5
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
 
-      - name: Install uv
-        uses: astral-sh/setup-uv@v4
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)